web/src/components/molecules/Common/Header/index.tsx

```diff
@@ -39,6 +39,10 @@ const HeaderMolecule: React.FC<Props> = ({
   const t = useT();
   const { logout } = useAuth();
   const navigate = useNavigate();
+  const url =
+    window.REEARTH_CONFIG?.editorUrl &&
+    currentWorkspace?.id &&
+    new URL(currentWorkspace.id, window.REEARTH_CONFIG?.editorUrl);
```

Comment:
```suggestion
  const url = useMemo(() => {
    if (window.REEARTH_CONFIG?.editorUrl && currentWorkspace?.id) {
      return new URL(currentWorkspace.id, window.REEARTH_CONFIG?.editorUrl);
    }
    return undefined;
  }, [currentWorkspace?.id]);
```

Reply:
What do you think about the path (`dashboard/`), which could change depending on the environment?
Should I code it?

Reply:
Okay, I'll fix it. Thank you!